typescript/cli/ci-test.sh

```diff
@@ -1,18 +1,47 @@
 #!/usr/bin/env bash
 
-# NOTE: This script is intended to be run from the root of the repo
-
-# the first arg to this script is a flag to enable the hook config as part of core deployment
-# motivation is to test both the bare bone deployment (included in the docs) and the deployment with the routing over igp hook (which is closer to production deployment)
-HOOK_FLAG=$1
-if [ -z "$HOOK_FLAG" ]; then
-  echo "Usage: fork.sh <hook>"
-  exit 1
+# set script location as repo root
```

Comment:
Sorry to say it but we're going to have big conflicts with this: https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/3305/files#diff-a873d0ddaa563d74371ca76d4a8660fabfef92b7c0cc6328057d6b1caf1fd74d
cc @nbayindirli

Reply:
I think conflict resolution should still be manageable though, as it could be easy enough to copy over the relevant parts into the functions you've got Noah